docs/asciidoc/http-server.adoc

```diff
@@ -339,10 +339,9 @@ include::{examplesdir}/sni/Application.java[lines=18..47]
 
 == HTTP Access Log
 
-The current logging support provides only the https://en.wikipedia.org/wiki/Common_Log_Format[Common Log Format].
+The `HTTP` access log could be enabled programmatically or by configuration. By default, it is disabled.
```

Comment:
Change the line to:

You can access the `HTTP` access either programmatically or by configuration. By default, it is disabled.